build/changelog-generator/templates/header.hbs

```diff
@@ -8,7 +8,10 @@
  <a name="{{version}}"></a>
 # {{version}} ({{date}})
 {{#if @root.linkCompare~}}
-  [Full Changelog]({{~@root.repoUrl}}/compare/{{previousTag}}...{{currentTag}})
+  [Full Changelog](
+    {{~#if @root.host}}{{~@root.host}}/{{/if~}}
+    {{~#if @root.owner ~}}{{@root.owner}}/{{/if~}}
```

Comment:
do all these newlines work? `yarn changelog`

Reply:
Looked fine when I tried it:
```md
 <a name="9.5.0"></a>
# 9.5.0 (2022-06-15)
[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v7.3.0...v9.5.0)

~~ TODO: https://chromiumdash.appspot.com/schedule ~~
We expect this release to ship in the DevTools of [Chrome XX](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.

## Notable Changes
~~ TODO: Move notable changes here ~~


## New Audits

* add work-during-interaction diagnostic ([#13982](https://github.com/GoogleChrome/lighthouse/pull/13982))
* add responsiveness metric for timespans ([#13917](https://github.com/GoogleChrome/lighthouse/pull/13917))
* lcp-lazy-loaded ([#12838](https://github.com/GoogleChrome/lighthouse/pull/12838))
* fr: uses-responsive-images-snapshot ([#12714](https://github.com/GoogleChrome/lighthouse/pull/12714))

```